### DISCUSSION TITLE
very slow to recieve emails

### DISCUSSION THREAD

**Discussion by rajeevrrs** [modoboa/modoboa#2700] [General]
# Impacted versions

* Distribution: Ubuntu 
* uname: `Linux mail.mydomain.com 5.15.0-1021-oracle modoboa/modoboa-installer#27-Ubuntu SMP Fri Oct 14 20:04:20 UTC 2022 aarch64 aarch64 aarch64 GNU/Linux`
* 
* Database: PostgreSQL 

# Problem

1. It takes around 30-55 min to receive a email from anywhere including Gmail, Yahoo, Outlook

2. I am able to send email but it arrives in spam folder of Gmail or Yahoo. 

3. How to edit and set custom SMTP
4. SPF Record - Unknown mechanism

**Comment by Spitfireap** [comment]
Hi, 

> 1. It takes around 30-55 min to receive a email from anywhere including Gmail, Yahoo, Outlook

This may be due to postscreen. You can check it by checking `/var/log/mail.log`. Please add if you installed modoboa with the installer or not.


> 2\. I am able to send email but it arrives in spam folder of Gmail or Yahoo.

Email is now complex. You might need to set up rDNS to ensure good deliverability. 

> 3\. How to edit and set custom SMTP

What do you mean exactly?


> 4\. SPF Record - Unknown mechanism

it seems like you corrected it.

**Comment by rajeevrrs** [comment]
@Spitfireap 

do spamassassin works without amavis & ClamAV
